lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -746,14 +746,14 @@ def check_jobs(self, job_ids, check_permission=True, projection=None):
             mongo_rec = job.to_mongo().to_dict()
             del mongo_rec['_id']
             mongo_rec['job_id'] = str(job.id)
-            mongo_rec['created'] = str(job.id.generation_time)
-            mongo_rec['updated'] = str(job.updated)
+            mongo_rec['created'] = job.id.generation_time.utcnow().replace(tzinfo=timezone.utc).isoformat()
```

Comment:
Can you just do   job.generation_time.isoformat() ? What is the the utcnow.replace for?